Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeError.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+/**
+ Error returned from the native PayPal flow
+ */
+@objc public enum BTPayPalNativeError: Int, LocalizedError {
+    /// Request is not of type BTPayPalNativeCheckoutRequest or BTPayPalNativeVaultRequest
+    case invalidRequest
+
+    /// Failed to fetch Braintree configuration
+    case fetchConfigurationFailed
+
+    /// PayPal is not enabled for this merchant in the Braintree Control Panel
+    case payPalNotEnabled
+
+    /// Could not find PayPal client ID in the Braintree configuration
+    case payPalClientIDNotFound
+
+    /// Invalid environment identifier found in the Braintree configuration
+    case invalidEnvironment
```

Comment:
Curious what we are considering an invalid environment here? In Android we are currently using the logic if `production` then send to `Live` environment, and everything else is `Sandbox`

Reply:
Yeah, we could do the same here. We _should_ only ever get back production or sandbox, so this error shouldn't ever occur. 🤷‍♀️ I don't really have a strong preference either way, but I'm happy to change this if you think defaulting to sandbox is better!

Reply:
I don't feel strongly but will just make sure that we are doing the same on Android and iOS. How would a merchant get configured for an environment other than production or sandbox? It might be a confusing error for them to try to resolve. But maybe it is best of error out if they aren't configured correctly.

Reply:
> How would a merchant get configured for an environment other than production or sandbox?

This should never happen -- but if it did happen, the merchant would need to reach out to Support and we'd need to investigate. Some of these error messages are really intended to help us debug more than to help merchants debug. So maybe it's better to keep it?